react/tic-tac-toe/src/app/screens/Game/components/Board/index.js

```diff
@@ -1,37 +1,38 @@
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
 
 import Square from '../Square';
 
 import styles from './styles.module.scss';
 
+const indexes = [0, 1, 2];
```

Comment:
You have to declare this const inside class or maybe declare it in a separated file name 'constants.js' and import it here.